examples/src/main/scala/org/apache/spark/examples/ml/TfIdfExample.scala

```diff
@@ -20,7 +20,7 @@ package org.apache.spark.examples.ml
 
 import org.apache.spark.{SparkConf, SparkContext}
 // $example on$
-import org.apache.spark.ml.feature.{HashingTF, IDF, Tokenizer}
```

Comment:
Why did we add CountVectorizer as an import here and not in JavaTfIdfExample? Since we aren't referencing it in code in either I'd probably leave it out of both personally (but either way consistency would be best).